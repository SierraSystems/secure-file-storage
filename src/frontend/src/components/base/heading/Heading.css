.head {
  padding-left: 40px;
  margin-right: 25px;
}

.sub-heading {
  padding-top: 35px;
  letter-spacing: 1.4px;
  text-transform: uppercase;
  font-size: 20px;
  font-weight: bold;
}

.text {
  padding-top: 15px;
  max-width: 90%;
}
