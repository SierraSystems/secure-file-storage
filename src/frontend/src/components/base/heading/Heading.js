import React from "react";
import "./Heading.css";
import "bootstrap/dist/css/bootstrap.css";

export default function Heading() {
  return (
    <div class="spacing">
      <h1 class="sub-heading head">Welcome to Secure File Upload & Storage</h1>
      <p class="text head">
        The Secure File Upload & Storage allows you to securely upload and store
        files on an S3 compatible storage bucket.
      </p>
    </div>
  );
}
