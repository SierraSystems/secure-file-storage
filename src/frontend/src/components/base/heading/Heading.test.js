import React from "react";
import Heading from "./Heading";
import testBasicSnapshot from "../../../TestHelper";

describe("Heading", () => {
  test("Component renders as expected", () => {
    testBasicSnapshot(<Heading />);
  });
});
