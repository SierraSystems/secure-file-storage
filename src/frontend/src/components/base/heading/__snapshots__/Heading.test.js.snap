// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Heading Component renders as expected 1`] = `
<div
  class="spacing"
>
  <h1
    class="sub-heading head"
  >
    Welcome to Secure File Upload & Storage
  </h1>
  <p
    class="text head"
  >
    The Secure File Upload & Storage allows you to securely upload and store files on an S3 compatible storage bucket.
  </p>
</div>
`;
